pyproject.toml

```
@@ -2,7 +2,7 @@
 requires = [
     "wheel",
     "setuptools<v60.0",
-    "cython>=0.28.0",
+    "cython>=0.28.0, <3.0",
```

理由不明になりそうなのでメモを書いておきましょう
```suggestion
    "cython>=0.28.0, <3.0", # NOTE: https://github.com/r9y9/pyopenjtalk/issues/55
```